downstream/modules/platform/proc-define-mesh-node-types.adoc

```diff
@@ -0,0 +1,154 @@
+[id="proc-define-mesh-node-types"]
+
+= Defining {AutomationMesh} node types
+
+To expand job capacity, create a standalone *execution node* that can be added to run alongside a deployment of {ControllerName}. 
+These execution nodes are not part of the {ControllerName} Kubernetes cluster. 
+The control nodes run in the cluster connect and submit work to the execution nodes through Receptor. 
+These execution nodes are registered in {ControllerName} as type `execution` instances, meaning they are only used to run jobs, not dispatch work or handle web requests as control nodes do.
+
+*Hop nodes* can be added to sit between the control plane of {ControllerName} and standalone execution nodes. 
+These hop nodes are not part of the Kubernetes cluster and are registered in {ControllerName} as an instance of type `hop`, meaning they only handle inbound and outbound traffic for otherwise unreachable nodes in different or more strict networks.
+
+The following procedure demonstrates how to set the node type for the hosts.
+
+.Procedure
+
+. From the navigation panel of {ControllerName}, select menu:Instances[].
+. On the *Instances* list page, click btn:[Add]. 
+The *Create new Instance* window opens.
++
+image:instances_create_new.png[Create new Instance]
```

Comment:
Should images in AsciiDoc have two colons as opposed to one? 
i.e. image::instances_create_new.png[Create new Instance]

Reply:
Flagged in a few places - sorry for the repetition